mix.exs

```diff
@@ -65,7 +69,9 @@ defmodule Ret.Mixfile do
       {:secure_random, "~> 0.5"},
       {:bamboo, "~> 1.3"},
       {:bamboo_smtp, "~> 1.7"},
-      {:guardian, "~> 1.2"},
+      {:guardian, "~> 2.1.1"},
```

Comment:
Updating Guardian to a release that includes an option to prevent halting in error handler, so that we can return error info in a graphql response

Context:
https://github.com/ueberauth/guardian/issues/401#issuecomment-367756347
https://github.com/ueberauth/guardian/releases/tag/2.1.0